Todo.py

```diff
@@ -0,0 +1,203 @@
+import discord
+from settings import *
+from checks import *
+
+TOKEN = "NTY4MDkyNTY5NTE5NzgzOTM2.XMiBDw.zCL0KhV_gOtxfyACctgYCLcLlEY"
+# PREFIX = "g:"
+
+# EMOJIS = {
+#     "wastebasket": "\U0001F5D1",
+#     "check": "\U00002705",
+#     "hourglass": "\U000023F3"
+# }
+
+#bot = commands.Bot(command_prefix=PREFIX)
+
+class Todo(commands.Cog):
+    """A suite of command to make a nice todo list."""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author == bot.user:
+            return
+        await bot.process_commands(message)
```

Comment:
you don't need to do this in a cog, not hte way the bot is built. So you can remove the listener.

Reply:
Same here, as you agreed I need you to delete this listener.

Reply:
I didn't psh yet but done

Reply:
Oh I see. Well please commit asap. It would prevent me from writing useless comments ;)